src/components/table/style.css

```diff
@@ -1,6 +1,7 @@
 :host {
 	display: table;
 	width: var(--table-width, 80%);
+	justify-content: flex-start;
```

Comment:
This rule is not available in a table display. 
![image](https://github.com/utily/smoothly/assets/25643315/100bc144-57e0-4264-9165-dda55b24a931)

Reply:
It is put there to be inherited by its children.